test/live_view_native/swiftui/rules_parser_test.exs

```diff
@@ -179,15 +182,7 @@ defmodule LiveViewNative.SwiftUI.RulesParserTest do
     test "parses complex modifier chains" do
       input = "color(color: .foo.bar.baz(1, 2).qux)"
 
-      output =
-        {:color, [],
-         [
-           {
-             :color,
-               {:., [],
-                [nil, {:., [], [:foo, {:., [], [:bar, {:., [], [{:baz, [], [1, 2]}, :qux]}]}]}]}
-           }
-         ]}
+      output = {:color, [], [color: {:., [], [{:., [], [{:., [], [{:., [], [nil, :foo]}, :bar]}, {:baz, [], [1, 2]}]}, :qux]}]}
```

Comment:
I think this is wrong, I'm looking into it....

Reply:
I believe this passes simply because elixir lists such as `[foo: 123]` is really `[{:foo, 123}]` but it should work for us either way